docs/source/user_guide/groups.rst

```diff
@@ -618,14 +618,16 @@ detections represented as |Detection| instances with their `label`, `location`,
     # Object label
     label = "vehicle"
 
-    # The center location ``(x, y, z)`` in camera coordinates of the bottom (y=0)
-    # face of the cuboid
+    # The center location of the object ``(x, y, z)`` in scene coordinates
+    # NOTE: the y coordinate will be offset by half the objects y size.
```

Comment:
This is essentially a bug. I think we should just fix this via: https://github.com/voxel51/fiftyone/pull/2376

Reply:
This was originally added to correct the positioning of cuboids in the kitti data set, but it should have been done via a setting, as the kitti labels are rotated.

Something like `settings['objectOriginStyle'] = 'centeredYFloor'`
or apply the offset to the labels themselves on import.